### PR TITLE
Added api/staff/all for the backend that lists every staff member

### PR DESCRIPTION
**Pull request by GeorgebigG**
For the staff I am adding api/staff/all that lists every staff member

https://organic-george.dokku-09.cs.ucsb.edu

### CODE REVIEW

**Review by JUstISAGE — APPROVED** [review]
LGTM

File: src/main/java/edu/ucsb/cs156/organic/controllers/StaffController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    @Operation(summary = "List all staff")
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN')")
+    @GetMapping("/all")
+    public Iterable<Staff> allCourses() {
+        User u = getCurrentUser().getUser();
+        log.info("u={}", u);
+
+        return staffRepository.findAll();
+    }
+}
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/organic/controllers/StaffController.java:50]: We actually don't want or need this route.

What we need is a route that returns all staff for a given course.

There is no use case for "all staff across all courses".

**Review by pconrad — CHANGES_REQUESTED** [review]
This is not useful in its current form.

An endpoint for getting all staff for a given course, though, would be useful.

**Comment by GeorgebigG** [comment]
Unfortunately I made a mistake with this pull request. I coded staff/all backend which apparently we didn't need

**Pull request closed without merging**